src/lazy_buffer.rs

```diff
@@ -16,8 +16,7 @@ where
         LazyBuffer {
             it,
             done: false,
-            buffer: Vec::new(),
-        }
+            buffer: Vec::new() }
```

Comment:
(Nitpick) Is this formatting change intentional?

Reply:
Oops, fixed!